src/main/java/com/qaprosoft/carina/demo/operations/Operation.java

```diff
@@ -0,0 +1,55 @@
+package com.qaprosoft.carina.demo.operations;
+
+import com.qaprosoft.carina.core.foundation.AbstractTest;
+import com.qaprosoft.carina.core.foundation.performance.Timer;
+import org.openqa.selenium.WebDriver;
+
+public class Operation {
```

Comment:
too bad example. Implementation should be much easier. Using a page declaration as private field is not good as on retry object became invalid with 90% probability.
Please refactor removing Operation class at all and providing as demo test just page object and testclass.
For example take a look to:
https://github.com/qaprosoft/carina/blob/5c6337466eda79e33cf167d44b1685957295cea6/carina-utils/src/test/java/com/qaprosoft/carina/core/foundation/performance/PerformanceTest.java